doc/ref_impl/cntt-ri/chapters/chapter03.md

```diff
@@ -23,45 +23,17 @@ Any virtual network functions and/or cloud-native network functions must choose
 
 `B/N/C <I opt> . <Flavour> . <S ext> . <A ext>`
 
-B/N<!---/C---> is used to specify the instance type (Basic, Network Intensive<!---, and Compute Intensive--->), different instance types are associated with different acceleartion extensions, network characteristics ([RM1: 4.2.4 Instance Types](../../../ref_model/chapters/chapter04.md#4.2.4)) and instance capabilities([RM1: 4.2.5 Instance Capabilities Mapping](../../../ref_model/chapters/chapter04.md#4.2.5)).
+B/N is used to specify the instance type (Basic, Network Intensive), different instance types are associated with different acceleartion extensions, network characteristics ([RM1: 4.2.4 Instance Types](../../../ref_model/chapters/chapter04.md#4.2.4)) and instance capabilities([RM1: 4.2.5 Instance Capabilities Mapping](../../../ref_model/chapters/chapter04.md#4.2.5)).
```

Comment:
while deleting comments, can we also correct "RM1" to "RM"?

Reply:
Thank you for the comment! I made the changes and updated the description of the PR with this information.